From 02c520c6fa44f9f2499c79e48531b59d62875a39 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Sat, 19 Sep 2015 09:58:39 -0700
Subject: [PATCH] Ticket 48266 - coverity -- unused variable 'init_retry'

Description: Backport error for Ticket 48266 - Fractional replication
             evaluates several times the same CSN
             (commit 05e127c89281cece8bc1fa79bac6b95cc23dcca9)
---
 ldap/servers/plugins/replication/repl5_tot_protocol.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/repl5_tot_protocol.c b/ldap/servers/plugins/replication/repl5_tot_protocol.c
index e004af4..7da893a 100644
--- a/ldap/servers/plugins/replication/repl5_tot_protocol.c
+++ b/ldap/servers/plugins/replication/repl5_tot_protocol.c
@@ -318,7 +318,6 @@ repl5_tot_run(Private_Repl_Protocol *prp)
 	int portnum = 0;
 	Slapi_DN *area_sdn = NULL;
 	CSN *remote_schema_csn = NULL;
-	int init_retry = 0;
 	Replica *replica;
 	ReplicaId rid = 0; /* Used to create the replica keep alive subentry */
 	
-- 
1.9.3

